plugins/sentinelone/komand_sentinelone/util/api.py

```diff
@@ -47,6 +46,39 @@ def clean(obj):
     return cleaned
 
 
+def set_agents_array(search: str, agent_details: str, agents: list) -> list:
```

Comment:
I suggest changing the `list` into `List[]` from typing with a proper typed values in the list. For example if it's a list of strings it will be `List[str]` - if it's a list of dict it will be for example `List[Dict[str, Any]]`. Import all those from `typing` library 😄